source/baseObject.py

```diff
@@ -1,24 +1,39 @@
 # A part of NonVisual Desktop Access (NVDA)
-# Copyright (C) 2007-2020 NV Access Limited, Christopher Toth, Babbage B.V., Julien Cochuyt
+# Copyright (C) 2007-2022 NV Access Limited, Christopher Toth, Babbage B.V., Julien Cochuyt
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
 
 """Contains the base classes that many of NVDA's classes such as NVDAObjects, virtualBuffers, appModules, synthDrivers inherit from. These base classes provide such things as auto properties, and methods and properties for scripting and key binding.
 """
 
+from typing import (
+	Any,
+	Callable,
+	Optional,
+	Set,
+	Union,
+)
 import weakref
 import garbageHandler
 from logHandler import log
 from abc import ABCMeta, abstractproperty
 
+GetterReturnT = Any
+GetterMethodT = Callable[["AutoPropertyObject"], GetterReturnT]

```

Comment:
I think `GetterReturnT` could be better supported with Generics, did you look into it? Regardless, the current approach is fine.

Reply:
I'm not sure what that should look like